### PR TITLE
Mobile v2.7 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by enahum — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md [hunk 1]
```diff
@@ -2,12 +2,41 @@
...
+
+### Compatibility
+ - **Upgrade to server version v7.8.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) v7.1.0 has ended and upgrading to server ESR v7.8.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Guest tags are now removed based on server configuration under **System Console > Authentication > Guest Access > Show Guest Tag**.
+ - Improved behavior of multi-request environments (such as HTTP1.1).
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:23]:
```suggestion
 - Improved behavior of multi-request environments (such as HTTP/1.1).
```

File: source/deploy/mobile-app-changelog.md [hunk 2]
```diff
@@ -2,12 +2,41 @@
...
+ - Added search result highlight that matches the words being searched.
+ - Added a new feature for remembering the last viewed channel or thread. It will help to jump back to the last channel or thread where the user left.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:25]:
```suggestion
 - Added a new feature for remembering the last viewed channel or thread that takes returning users to the last channel or thread they visited.
```

**Review by cwarnermm — APPROVED** [review]
Provided feedback inline. Approving for expediency.

File: source/deploy/mobile-app-changelog.md [hunk 3]
```diff
@@ -2,12 +2,41 @@
...
+ - Reduced the padding/margins on the **Files** tab in the **Search** page.
```
Comment (larkox) [inline source/deploy/mobile-app-changelog.md:26]: This is not exactly what the PR finally did. We can leave it as something like:
```suggestion
 - Improve the looks on the search results.
```

**Review by larkox — APPROVED** [review]
Just one change. Approving preemptively.

**Pull request merged**